packages/flutter/lib/src/rendering/sliver_multi_box_adaptor.dart

```diff
@@ -75,6 +75,17 @@ abstract class RenderSliverBoxChildManager {
   /// the child list after this function returns.
   void didAdoptChild(RenderBox child);
 
+  /// Called during layout to indicate whether this object did not provide
+  /// enough children for the [RenderSliverMultiBoxAdaptor] to fill the
+  /// [SliverConstraints.remainingPaintExtent].
```

Comment:
Went with "provided insufficient children" because it's not really a failure on the part of this object if it doesn't have more children.